src/k8s/pkg/k8sd/database/schema.go

```diff
@@ -18,9 +18,11 @@ var (
 	SchemaExtensions = []schema.Update{
 		schemaApplyMigration("kubernetes-auth-tokens", "000-create.sql"),
 		schemaApplyMigration("cluster-configs", "000-create.sql"),
+		schemaApplyMigration("worker-nodes", "000-create.sql"),
 		schemaApplyMigration("worker-tokens", "000-create.sql"),
 		schemaApplyMigration("feature-status", "000-feature-status.sql"),
 		schemaApplyMigration("worker-tokens", "001-add-expiry.sql"),
+		schemaApplyMigration("worker-nodes", "002-delete.sql"),
```

Comment:
it sounds like the order and names of these migrations is important -- but we've just got bare strings identifying them.  Also, are they lexicographically ordered or numerically ordered?  I'm worried about the day we run out of numbers (999).  

```
⚡ ls -al
total 8
drwxrwxr-x  2 addyess addyess 4096 Sep 20 13:14 .
drwxrwxr-x 12 addyess addyess 4096 Sep 20 13:13 ..
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:13 0001-stuff
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:13 001-stuff
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:14 0090-stuff
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:14 009-stuff
```

Reply:
the alternative i'm hoping for:

```
⚡ ls -alv
total 8
drwxrwxr-x  2 addyess addyess 4096 Sep 20 13:14 .
drwxrwxr-x 12 addyess addyess 4096 Sep 20 13:13 ..
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:13 0001-stuff
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:13 001-stuff
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:14 009-stuff
-rw-rw-r--  1 addyess addyess    0 Sep 20 13:14 0090-stuff
```

Reply:
Currently, the order that microcluster uses is simply the order of schema migrations in this struct. Unfortunately, we started naming the migrations with these numbers -they don't mean anything right now. 

We can drop the numbers all-together if that makes things less messy. However, I don't like the idea of changing existing migration names- if microcluster migrations decide to use the names in the future this could get us into trouble (The migration names are stored in a table).

Reply:
thanks so much 👍🏻